File: server/sequelize/models/Company.js
import DataTypes from "sequelize";

/**
 * Modelo de 'empresa' en base de datos
 * @param {*} sequelize 
 */
export const companyModel = (sequelize) => {
  sequelize.define(
    "Company",
    {
      companyId: {
        type: DataTypes.INTEGER,
        primaryKey: true,
        autoIncrement: true,
        field: "id_empresa",
      },
      name: {
        type: DataTypes.STRING,
        allowNull: false,
        field: "nombre",
      },
    },
    {
      tableName: "empresa",
      timestamps: false,
    }
  );
};

File: mi_conjunto_residentes/lib/models/company.dart

/// Modelo de 'empresa'
class Company {
  int companyId;
  String name;

  Company({this.companyId, this.name});

  factory Company.fromJson(Map<String, dynamic> json) {
    return Company(companyId: json["companyId"], name: json["name"]);
  }
}
